core/src/main/scala/com/lightbend/paradox/markdown/AnchorLink.scala

```diff
@@ -28,7 +28,7 @@ import scala.collection.JavaConverters._
 class AnchorLinkSerializer extends ToHtmlSerializerPlugin {
   def visit(node: Node, visitor: Visitor, printer: Printer): Boolean = node match {
     case anchor: AnchorLinkSuperNode =>
-      printer.print(s"""<a href="#${anchor.name}" name="${anchor.name}" class="anchor"><span class="anchor-link"></span></a>""")
+      printer.print(s"""<a href="#${anchor.name}" name="${anchor.name}" class="anchor"><span class="anchor-link icon"></span></a>""")
```

Comment:
Is this necessary? It doesn't look related to snippets and the `ancher-link` class is already pretty specific.

Reply:
I extracted the `.icon` CSS class to be general so it could be reused here and on the snippet source links.

Should I have done it differently?

Reply:
The properties can also be shared by listing the selectors:

```css
a.anchor .anchor-link, .icon {
    /* ... */
}
```

Reply:
Fair enough. Pushed a commit that reverts the changes to the scripted tests and now shares properties across selectors.